src/components/UI/Slider/SliderWithMarkers.vue

```diff
@@ -113,7 +117,8 @@ export default {
     },
     markerSliderSelected(markerIndex) {
       // invoked when a marker has been selected
-      this.activeMarkerIndex = markerIndex;
+      if (!this.isDragDisabled)
+        this.activeMarkerIndex = markerIndex;
```

Comment:
this leaves the markers undraggable but it doesn't affect the functionality of clicking the markers. they can still be clicked and the question will popup